Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "finality-grandpa"
-version = "0.14.2"
+version = "0.15.0"
```

Comment:
Just bumping to `0.14.3` should suffice, there's no breaking change introduced here right?